Makefile

```diff
@@ -9,7 +9,7 @@ HELM_VERSION ?= v3.11.2
 CHART_APPVERSION ?= v0.7.0 # x-release-please-version
 
 # renovate: datasource=docker depName=cytopia/yamllint
-YAMLLINT_VERSION ?= alpine-1-0.14
+YAMLLINT_VERSION ?= alpine
```

Comment:
i removed the version tag, because that version any other pinned version i tried seem to not be up to date and point to and older version of YAML lint that doesn't work